docs/rst/developer_manual/installation/sources/linux.rst

```diff
@@ -4,6 +4,306 @@
+
+        .. code-block:: bash
+
+            cd ~/DDS-Visualizer-Plugin
+            git clone https://github.com/eProsima/foonathan_memory_vendor.git
+            mkdir foonathan_memory_vendor/build
+            cd foonathan_memory_vendor/build
+            cmake .. -DCMAKE_INSTALL_PREFIX=~/DDS-Visualizer-Plugin/install -DBUILD_SHARED_LIBS=ON
+            cmake --build . --target install
+
+    * `Fast CDR <https://github.com/eProsima/Fast-CDR.git>`_
+
+        .. code-block:: bash
+
+            cd ~/DDS-Visualizer-Plugin
+            git clone https://github.com/eProsima/Fast-CDR.git
+            mkdir Fast-CDR/build
+            cd Fast-CDR/build
+            cmake .. -DCMAKE_INSTALL_PREFIX=~/DDS-Visualizer-Plugin/install
+            cmake --build . --target install
+
+    * `Fast DDS <https://github.com/eProsima/Fast-DDS.git>`_
+
+        .. code-block:: bash
+
+            cd ~/DDS-Visualizer-Plugin
+            git clone https://github.com/eProsima/Fast-DDS.git
+            mkdir Fast-DDS/build
+            cd Fast-DDS/build
+            cmake .. -DCMAKE_INSTALL_PREFIX=~/DDS-Visualizer-Plugin/install -DCMAKE_PREFIX_PATH=~/DDS-Visualizer-Plugin/install
+            cmake --build . --target install
+
+#.  Once all dependencies are installed, install *eProsima DDS Visualizer Plugin*:
+
+    .. code-block:: bash
+
+        cd ~/DDS-Visualizer-Plugin
+        git clone https://github.com/eProsima/plotjuggler-fastdds-plugins.git
+        mkdir DDS-Visualizer-Plugin/build
+        cd DDS-Visualizer-Plugin/build
+        cmake .. \
+            -DCMAKE_INSTALL_PREFIX=~/DDS-Visualizer-Plugin/install \
+            -DCMAKE_PREFIX_PATH=~/DDS-Visualizer-Plugin/install \
+            -DQT_PATH=<qt-installation-path>
+        cmake --build . --target install
+
+
+.. note::
+
+    By default, *eProsima DDS Visualizer Plugin* does not compile tests.
+    However, they can be activated by downloading and installing `Gtest <https://github.com/google/googletest>`_
+    and building with CMake option ``-DBUILD_TESTS=ON``.
+
+
+.. _global_installation_sl:
+
+Global installation
+-------------------
+
+To install *eProsima Fast DDS* system-wide instead of locally, remove all the flags that
+appear in the configuration steps of :code:`Fast-CDR`, :code:`Fast-DDS`, and
+:code:`DDS-Visualizer-Plugin`, and change the first in the configuration step of :code:`foonathan_memory_vendor` to the
+following:
+
+.. code-block:: bash
+
+    -DCMAKE_INSTALL_PREFIX=/usr/local/ -DBUILD_SHARED_LIBS=ON
+
+.. _run_app_colcon_sl:
+
+Run an application
+==================
+
+To run the *eProsima DDS Visualizer Plugin* application, source the *Fast DDS* libraries
+and execute the executable file that has been installed in :code:`<install-path>/dds_visualizer/bin/dds_visualizer`:
+
+.. code-block:: bash
+
+    # If built has been done using colcon, all projects could be sourced as follows
+    source install/setup.bash
+    ./<install-path>/dds_visualizer/bin/dds_visualizer
+
```

Comment:
Add a warning saying that if the plugin does not load properly or app crashes it is because source of `fastdds` has not been done